wordpress-mobile/AztecEditor-Android
# Issue/update format icons

### Fix
Update format toolbar button icons as described in https://github.com/wordpress-mobile/AztecEditor-Android/issues/351, https://github.com/wordpress-mobile/AztecEditor-Android/issues/352, and https://github.com/wordpress-mobile/AztecEditor-Android/issues/378.

### Test
1. Open demonstration app.
2. Notice updated format toolbar button icons.
